docs/modules/operation/pages/deep-dive/service-assurance/configuration.adoc

```diff
@@ -1,40 +1,40 @@
 
 [[ga-pollerd-configuration]]
-= Pollerd Configuration
+= Configure Pollerd
 
-.Configuration and log files related to Pollerd.
+.Configuration and log files related to Pollerd
```

Comment:
This table seems redundant to the pollerd daemon page in the reference section.

Reply:
I'll update and link.